cmd/admin-handlers.go

```diff
@@ -1504,7 +1508,7 @@ func (a adminAPIHandlers) TraceHandler(w http.ResponseWriter, r *http.Request) {
 	trcErr := r.URL.Query().Get("err") == "true"
 
 	// Validate request signature.
-	adminAPIErr := checkAdminRequestAuthType(ctx, r, "")
+	_, adminAPIErr := checkAdminRequestAuthType(ctx, r, iampolicy.ListServerInfoAdminAction, "")
```

Comment:
should this be some other action?

Reply:
Trace and console messages are still server info type permissions, hence leaving this as it is - don't see a case for making this permission more specific.